### DISCUSSION TITLE
Sass Deprecation Warning issue relapse

### DISCUSSION THREAD

**Discussion by cshen-dev** [cotes2020/jekyll-theme-chirpy#2018] [Q&A] [answered]
### Checklist

- [X] I have read and accepted the [contributing guidelines](https://github.com/cotes2020/jekyll-theme-chirpy/blob/master/docs/CONTRIBUTING.md).

### How did you create the site?

Generated from `chirpy-starter`

### Description

I can still see Deprecation Warning: Sass @import issue with the latest version of chirpy 7.1.1
While I do see similar issue was discussed and resolved below
https://github.com/cotes2020/jekyll-theme-chirpy/issues/1875

### Operations you have already tried

I create a new repo using chirpy-starter and I can reproduce the Deprecation Warning issue.

### Anything else?

I'm curious if I'm the only one who see this problem

**Comment by kungfux** [comment, marked as answer]
That's fine. Tools get updated from time to time. This deprecation is just information for developers to look out for, so they're ready when it's time to upgrade.

```
Deprecation Warning: Sass @import rules are deprecated and will be removed in Dart Sass 3.0.0.
```
